waffle-chai/src/index.ts

```diff
@@ -19,4 +21,6 @@ export function waffleChai(chai: Chai.ChaiStatic, utils: Chai.ChaiUtils) {
   supportProperHex(chai.Assertion);
   supportChangeBalance(chai.Assertion);
   supportChangeBalances(chai.Assertion);
+  supportEthCalled(chai.Assertion);
+  supportEthCalledWith(chai.Assertion);
```

Comment:
I would consider having those two in the same file. We could then extract the common parts into a function, for example for data type checking